.github/workflows/sync.yml

```
@@ -3,7 +3,8 @@ name: Version Sync
 on:
   workflow_dispatch:
   schedule:
-    - cron: "0 0 * * *"
+    # Run once a week at 00:05 AM UTC on Sunday.
```

```suggestion
    # Run once a week at 00:05 UTC on Sunday.
```
There is no AM or PM with the 24-hour clock